src/main/java/com/nasrpi/filesharing/UploadFileResponse.java

```diff
@@ -0,0 +1,57 @@
+package com.nasrpi.filesharing;
```

Comment:
is this not a request/response model for upload file? it would be better if we use it that way.

Reply:
didn't understand! can you explain it again what exactly is expected?

Reply:
done